jsonnet/kube-events-exporter/kube-events-exporter.libsonnet

```diff
@@ -0,0 +1,104 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+
+{
+  local kee = self,
+
+  config:: {
+    namespace: error 'must provide namespace',
+    version: error 'must provide version',
+    image: error 'must provide image',
+
+    commonLabels: {
+      'app.kubernetes.io/name': 'kube-events-exporter',
+      'app.kubernetes.io/version': kee.config.version,
+      'app.kubernetes.io/component': 'events-exporter',
+    },
+
+    selectorLabels: {
+      [labelName]: kee.config.commonLabels[labelName]
+      for labelName in std.objectFields(kee.config.commonLabels)
+      if !std.setMember(labelName, ['app.kubernetes.io/version'])
+    },
+  },
+
+  kubeEventsExporter+:: {
+    clusterRoleBinding:
+      local clusterRoleBinding = k.rbac.v1.clusterRoleBinding;
+
+      clusterRoleBinding.new() +
+      clusterRoleBinding.mixin.metadata.withLabels(kee.config.commonLabels) +
+      clusterRoleBinding.mixin.metadata.withName('kube-events-exporter') +
+      clusterRoleBinding.mixin.roleRef.withApiGroup('rbac.authorization.k8s.io') +
+      clusterRoleBinding.mixin.roleRef.withName('kube-events-exporter') +
+      clusterRoleBinding.mixin.roleRef.mixinInstance({ kind: 'ClusterRole' }) +
+      clusterRoleBinding.withSubjects([{
+        kind: 'ServiceAccount',
+        name: 'kube-events-exporter',
```

Comment:
best to reference `kee.serviceAccount.metadata.name` instead, that way it automatically gets changed here if the serviceaccount name changes for whatever reason